src/resources/ProductListingConfiguration/ProductListingConfigurationInterfaces.ts

```diff
@@ -0,0 +1,248 @@
+import {DeprecatedPaginated} from '../InternalBaseInterface.js';
+import {
+    FacetRequestOptions,
+    FieldOperatorType,
+    FieldValueType,
+    FilterTypeEnum,
+    RankingTypeEnum,
+    RestFacetRequest,
+} from '../index.js';
+
+export const isRankingTypeEnum = (value: unknown): value is RankingTypeEnum =>
+    value === RankingTypeEnum.BOOST || value === RankingTypeEnum.BURY;
+
+export const isFilterTypeEnum = (value: unknown): value is FilterTypeEnum =>
+    value === FilterTypeEnum.INCLUDE || value === FilterTypeEnum.EXCLUDE;
+
+export enum SortOrderEnum {
+    ASC = 'ASC',
+    DESC = 'DESC',
+}
```

Comment:
i think these have changed recently in the commerce-hub

Reply:
I'll take a look